runtime/runtime-params-estimator/src/main.rs

```diff
@@ -176,6 +189,9 @@ fn main_docker(state_dump_path: &Path) -> anyhow::Result<()> {
 
         let mut buf = String::new();
         buf.push_str("set -ex;\n");
+        buf.push_str("export CARGO_PROFILE_RELEASE_LTO CARGO_PROFILE_RELEASE_CODEGEN_UNITS;\n");
+        buf.push_str("echo $CARGO_PROFILE_RELEASE_LTO;\n");
+        buf.push_str("echo $CARGO_PROFILE_RELEASE_CODEGEN_UNITS;\n");
```

Comment:
This solves the right problem, but I think there's a better way to do this.

The problem with this setup is that pass-thru requires the person who uses the stuff to a) know about LTO b) to manually set it up. A better meta approach is to encode such tribal as LTO as code. So here I suggest to do the following:

Adding a `--full` flag to the estimator's CLI with the semantics of "run the estimation in the fully production setting to get usable cost table". 

If the `--full` is passed, this script just *sets* lto, instead of passing it through. 

PS: don't really like the `full` name, but, having spend 2 minutes looking for alternatives, I don't know a better one. SUggestions welcome!